pandas/tests/plotting/test_datetimelike.py

```diff
@@ -209,6 +210,18 @@ def test_line_plot_period_series(self):
         for s in self.period_ser:
             _check_plot_works(s.plot, s.index.freq)
 
+    @pytest.mark.slow
+    def test_line_plot_period_mlt_series(self):
+        # test period index line plot for series with multiples (`mlt`) of the
+        # frequency rule code
+        multiplier = [1, 9]  # multiplier for freq.rule_code
```

Comment:
you can parameterize over this

Reply:
Like this?
```
@pytest.mark.slow
@pytest.mark.parametrize('freqs', self.freq)
def test_line_plot_period_mlt_series(self, freqs):
    # test period index line plot for series with multiples (`mlt`) of the
    # frequency rule code. tests resolution of issue #14763
    multiplier = [1, 9]  # multiplier for freq.rule_code
    for mult in multiplier:
        freq_code = str(mult) + freqs  # construct frequency code
        idx = period_range('12/31/1999', freq=freq_code, periods=100)
        s = Series(np.random.randn(len(idx)), idx)
        _check_plot_works(s.plot, s.index.freq.rule_code)
```
Or do you want me to check other dtypes?

Reply:
Possibly parametrize over both frees and multiple (two separate ones).

Do we need to test each of these frequencies? Do we expect them to take different code paths?

Reply:
As far as I know this means that I have to explicitly pass all combinations to `parametrize`:
```@pytest.mark.parametrize('freqs,multipliers', [('S', 1), ('H', 1), ('W', 1), ('M', 1), ('S', 6), ('H', 6), ('W', 6), ('M', 6)])```
Or is there any other (implicit) way to feed combinations to `parametrize`? Using `itertools.product` in the decorator doesn't seem like a good solution.

Yep, I agree. I removed some freqs.. I Already thought about removing some freqs, but my "insight" and knowledge of the pandas code paths are too shallow, so I decided to leave the freqs proposed by the existing tests.

Reply:
I explicitly parametrized the frequencies, including one check for the freq rule code "multiplied" with 1 and omitting the quarter-check. The method doesn't contain any more references to its class instance, thus could be a function. But looking at other methods in `test_datetimelike.py` this seems to be ok.